testing/test_dataset.py

```diff
@@ -136,9 +136,9 @@ def XformS5E25D40Block0s():
     return {8: np.array([[9.543397173413723, 11.867961374296353, 13.912108941026776],
                         [9.552495679176754, 11.875262229083035, 13.92243287008154],
                         [9.557954782634571, 11.881650477021381, 13.929659620419873]]),
-            80: np.array([[-0.112179830308469, 0.11153442820188236, -1.0917475374833345],
-                          [-0.06482956124755412, 0.1280639923140157, -0.9732897698356204],
-                          [-0.05541021462917799, 0.13618300377160736, -0.9656043640863767]]),
+            80: np.array([[-0.112179830308469, 0.030588803898888246, -1.0917475374833345],
+                          [-0.06482956124755412,  0.0387078153564799, -0.9732897698356204],
+                          [-0.05541021462917799, 0.05523737946861323, -0.9656043640863767]]),
```

Comment:
No sorry- I just wanted the tests to hold valid if not every subchannel out of a transformed channel was given a negative gain. These Y min,mean,max values were just reverted back to the original values pre-all transforms, so that the distinction is clear btwn flipping the entire channel's MMM array and flipping MMM's by subchannel.